django/contrib/auth/admin.py

```diff
@@ -176,7 +178,8 @@ def user_change_password(self, request, id, form_url=''):
 
         request.current_app = self.admin_site.name
 
-        return TemplateResponse(request,
+        return TemplateResponse(
+            request,
```

Comment:
Oh bummer! I'm so used to move stuff around until all the warning are gone, haha.

I'll rever this change right now, thanks!